src/index.html

```diff
@@ -1,13 +1,61 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <link href="https://fonts.googleapis.com/css2?family=Comforter+Brush&family=Roboto:wght@300;400;500&display=swap" rel="stylesheet">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="cards" data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="imac A1419"
+        class="cards__image"
+      >
+
+      <div class="cards__description">
+        <div class="cards__content">
+          <div class="cards__title">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+          </div>
+          <div class="cards__code">
+            Код товара: 195434
+          </div>
+
+          <div class="cards__rang">
+            <div class="cards__img">
+              <div class="cards__star_4">
+                <div class="cards__star"></div>
+                <div class="cards__star"></div>
+                <div class="cards__star"></div>
+                <div class="cards__star"></div>
+                <div class="cards__star"></div>
+              </div>
+            </div>
+            <div class="cards__feedback">
+              Отзывов: 5
+            </div>
+          </div>
+
+          <div class="cards__prise-block">
+            <div class="cards__prise-text">Цена:</div>
+            <div class="cards__prise">69 999 грн</div>
+          </div>
+
+          <a
+          href="#"
+          class="cards__cards_buy"
+          data-qa="hover"
+          >
+            Купить
+          </a>
```

Comment:
```suggestion
          <a
            href="#"
            class="cards__cards_buy"
            data-qa="hover"
          >
            Купить
          </a>
```
check your code style